UPGRADE-3.3.md

```diff
@@ -244,6 +244,18 @@ FrameworkBundle
    class has been deprecated and will be removed in 4.0. Use the
    `Symfony\Component\Workflow\DependencyInjection\ValidateWorkflowsPass` class instead.
 
+ * The `Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\TranslationDumperPass`
+   class has been deprecated and will be removed in 4.0. Use the
```

Comment:
this will have to go in a new UPGRADE-3.4.md file actually. It is too late to include this in 3.3 as we already released the beta